Tests/Unit/Domain/Index/TcaIndexer/TcaTableServiceTest.php

```diff
@@ -141,16 +161,16 @@ public function allConfiguredAndAllowedTcaColumnsAreReturnedAsFields()
         );
         $this->inject($subject, 'logger', $this->getMockedLogger());
 
-        $this->assertSame(
```

Comment:
Remove or keep - you decide. But we usually don't keep these commented code lines.

Reply:
I've marked tests as incomplete and reverted to comment code out.